### DISCUSSION TITLE
I cant for the life of me figure out how to attach a file in banana cake pop

### DISCUSSION THREAD

**Discussion by onionhammer** [ChilliCream/hotchocolate#5062] [General]
Maybe I'm dumb, but I've been reading the doc on uploading a file via hotchocolate and have that configured, but attaching the file and uploading it into a variable w/ banana cake pop doesnt seem so straighforward.

![image](https://user-images.githubusercontent.com/969938/168303477-a7318220-48f9-4f4f-9deb-7b14a3492f21.png)

When I click "Save" on this dialog nothing appears to happen, the dialog stays open

How do I reference the file in the variables (if indeed it is attaching)?

**Comment by PascalSenn** [comment]
Once you have added the file you can reference it in the variables via strings on upload inputs:
<img width="352" alt="image" src="https://user-images.githubusercontent.com/14233220/168324549-172a5229-7357-4b58-ad17-f654f871ad45.png">
<img width="668" alt="image" src="https://user-images.githubusercontent.com/14233220/168324729-92096d0d-3116-4791-9ab4-6f91dc2e651d.png">

**Reply by onionhammer** [reply]
Ahh that works when I add the file via the 'Documents' pane, but not when I add the document via this button
![image](https://user-images.githubusercontent.com/969938/168391546-9398fdfa-dc69-4856-b93f-99d14f371e3a.png)

Using that button the 'Save' dialog doesnt appear to work properly

**Comment by kmcclellan** [comment]
Possibly related: #6143
